src/pages/CareList.tsx

```diff
@@ -59,77 +165,392 @@ const CareList = () => {
-                <List.Item.Meta
-                  title={
-                    <div className="pretty-wrapper">
-                      {t("care.fromTo", { from: care.careStart, to: care.careEnd, days: care.numberOfDays })}
-                    </div>
-                  }
-                  description={
-                    <Descriptions
-                      bordered
-                      column={{ xs: 1, sm: 1, md: 2, lg: 2, xl: 2, xxl: 2 }}
-                      size="small"
-                    >
-                      <Descriptions.Item label={t("animalType")}>
-                        {t(`animalTypes.${care.animalType}`)}
-                      </Descriptions.Item>
-                      <Descriptions.Item label={t("totalPrice")}>
-                        {care.totalPrice}
-                      </Descriptions.Item>
-                      <Descriptions.Item label={t("caretaker")}>
-                        <UserInfoPill user={care.caretaker} isLink={true} />          
-                      </Descriptions.Item>
-                      <Descriptions.Item label={t("client")}>
-                        <UserInfoPill user={care.client} isLink={false} />
-                      </Descriptions.Item>
-                      <Descriptions.Item label={t("care.currentStatus")}>
-                        {care.getCurrentStatusText()}
-                      </Descriptions.Item>
-                    </Descriptions>
+                <Select
+                  notFoundContent={t("noData")}
+                  showSearch
+                  allowClear
+                  mode="multiple"
+                  placeholder={t(`careSearch.${filterName}`)}
+                  filterOption={(searchedInput, option) =>
+                    (option?.label ?? "")
+                      .toLowerCase()
+                      .includes(searchedInput.toLowerCase())
                   }
+                  options={emails.map((email) => ({
+                    value: email,
+                    label: email,
+                  }))}
+                  onChange={(value) => onEmailsChanged(value)}
                 />
-              </List.Item>
-            </Badge.Ribbon>
-          )}
+              </Tooltip>
+            </div>
+          );
+
+        default:
+          return null;
+      }
+    });
+  };
+  //
+
+  return (
+    <div className="care-list-main-container">
+      <div className="care-list-sidebar">
+        <h2>{t("sort.title")}</h2>
+        <Select
+          style={{ width: "100%" }}
+          showSearch={false}
+          value={pagingParams.sortBy}
+          defaultValue="submittedAt"
+          options={[
+            "id",
+            "submittedAt",
+            "caretakerStatus",
+            "clientStatus",
+            "careStart",
+            "careEnd",
+            "description",
+            "dailyPrice",
+            "animal_animalType",
+            "selectedOptions",
+            "caretaker_email",
+            "client_email",
```

Comment:
Myśle że nie ma sensu dawać możliwości sortowania po `id`, `description`i `selectedOptions`.  
`id` jest bardziej dla nas na potrzeby bazy danych i do identyfikowania opiek, a sortowanie po opisie  i selectedOptions - to raczej zbędne z punktu widzenia użytkownika.
```suggestion
          options={[
            "submittedAt",
            "caretakerStatus",
            "clientStatus",
            "careStart",
            "careEnd",
            "dailyPrice",
            "animal_animalType",
            "caretaker_email",
            "client_email",
```